src/test/java/com/google/api/codegen/testdata/php_library.baseline

```diff
@@ -55,59 +68,22 @@ class LibraryServiceApi
     const _CODEGEN_NAME = 'GAPIC';
     const _CODEGEN_VERSION = '0.0.0';
 
-    private static $grpcBootstrap;
     private static $shelfNameTemplate;
     private static $bookNameTemplate;
-    private static $pageStreamingDescriptors;
 
+    private $grpcBootstrap;
     private $stub;
     private $scopes;
     private $defaultCallSettings;
     private $descriptors;
 
     /**
-     * This should not be called outside of the implementation file.
-     */
-    static function initClass()
-    {
-        if (!empty(self::$grpcBootstrap)) {
-            throw new RuntimeException("PublisherApi::initClass called more than once");
-        }
-
-        self::$grpcBootstrap = GrpcBootstrap::defaultInstance();
-
-        self::$shelfNameTemplate = new PathTemplate("shelves/{shelf}");
-        self::$bookNameTemplate = new PathTemplate("shelves/{shelf}/books/{book}");
-
-        $listShelvesPageStreamingDescriptor =
-                new PageStreamingDescriptor([
-                    'requestPageTokenField' => 'page_token',
-                    'responsePageTokenField' => 'next_page_token',
-                    'resourceField' => 'shelves']);
-        $listBooksPageStreamingDescriptor =
-                new PageStreamingDescriptor([
-                    'requestPageTokenField' => 'page_token',
-                    'responsePageTokenField' => 'next_page_token',
-                    'resourceField' => 'books']);
-        $listStringsPageStreamingDescriptor =
-                new PageStreamingDescriptor([
-                    'requestPageTokenField' => 'page_token',
-                    'responsePageTokenField' => 'next_page_token',
-                    'resourceField' => 'strings']);
-        self::$pageStreamingDescriptors = [
-            'listShelves' => $listShelvesPageStreamingDescriptor,
-            'listBooks' => $listBooksPageStreamingDescriptor,
-            'listStrings' => $listStringsPageStreamingDescriptor
-        ];
-    }
-
-    /**
      * Formats a string containing the fully-qualified path to represent
      * a shelf resource.
      */
     public static function formatShelfName($shelf)
```

Comment:
This was already covered in a separate discussion - we'll leave this as is for now, and have a broader discussion across languages.